pkg/parse/validator_test.go

```diff
@@ -240,22 +183,18 @@ func TestFix(t *testing.T) {
 		return resultUBP, resultBP
 	}
 
-	diffUBP := func(t *testing.T, str string, ubp1, ubp2 any, diffFile string) {
-		t.Run("DiffUBP", func(t *testing.T) {
-			if ubp1 == nil || ubp2 == nil {
-				t.Fatal("UBP objects are nil, cannot diff")
+	diffObjects := func(t *testing.T, s1, s2 any, diffFile string) {
+		t.Run("DiffObj", func(t *testing.T) {
+			if s1 == nil || s2 == nil {
+				t.Fatal("objects are nil, cannot diff")
```

Comment:
I renamed the helper anonymous function to `diffObjects` because it is universal.